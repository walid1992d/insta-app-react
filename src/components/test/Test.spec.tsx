
import React from 'react';
import {TestComponent} from './Test';
import {shallow} from 'enzyme';
describe("TestComponent", ()=> {
    let props: any;
    let getShallowComponent:Function;

    beforeEach(()=> {
        props = {
            text: 'Hello Test'
        }
        getShallowComponent = ()=> shallow(<TestComponent {...props} ></TestComponent>);
    })

    it('should render', () => {
        expect(getShallowComponent()).toMatchSnapshot();
    })
})
