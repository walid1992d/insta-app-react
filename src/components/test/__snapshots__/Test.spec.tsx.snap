// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent should render 1`] = `
<div
  className="TestComponent"
>
  Hello Test
</div>
`;
